### PR TITLE
Use pre-generated local private keys for default AMD keys

### PR DESCRIPTION
**Pull request by yawangwang**
Load local ECDSA private keys instead of using `ecdsa.GenerateKey` because the underlying implementations will cause our internal test environment crash.

### CODE REVIEW

**Comment by deeglaze** [comment]
Have you verified that these changes stop your test from crashing?

File: testing/fake_certs.go [hunk 1]
```diff
@@ -319,16 +319,12 @@ func DefaultAsvk() (*rsa.PrivateKey, error) {
 
 // DefaultVcek returns a new ECDSA key on the expected curve for a VCEK.
 func DefaultVcek() (*ecdsa.PrivateKey, error) {
-	privateKey, err := ecdsa.GenerateKey(elliptic.P384(), insecureRandomness)
```
Comment (deeglaze) [inline testing/fake_certs.go:322]: Would you also create pre-generated default ask, asvk, and ark private keys? That would make the test suites run a lot faster I think.

Comment (yawangwang) [reply testing/fake_certs.go:322]: Done.

**Comment by yawangwang** [comment]
> Have you verified that these changes stop your test from crashing?

Though those changes cannot stop my test from crashing, it might be a good practice to use the pre-generated test private keys to make tests run faster.

File: testing/data/data.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 Google LLC
```
Comment (deeglaze) [inline testing/data/data.go:1]:
```suggestion
// Copyright 2024 Google LLC
```

**Review by deeglaze — APPROVED** [review]

File: testing/data/data.go (hunk 2 above)
Comment (yawangwang) [reply testing/data/data.go]: Done.

**Comment by yawangwang** [comment]
ooops looks like the PR merging process is a bit different from go-tpm-tools, sending 1 reapproval request to @deeglaze

**Review by deeglaze — APPROVED** [review]

**Pull request merged**